lib/Doctrine/ORM/EntityManager.php

```diff
@@ -145,6 +146,13 @@
     /** @var Cache The second level cache regions API. */
     private $cache;
 
+    /**
+     * Flag shows that flush in progress
```

Comment:
Either "This flag shows that flush is in progress" or "Flag that shows that flush is in progress"… or maybe nothing, this comment is a bit tautological

Reply:
As you wish, I don't know what the current stance is towards this :)